pom.xml

```diff
@@ -8,7 +8,7 @@
     <name>Exasol Row Level Security</name>
     <description>This projects provides a plug-in to the Exasol database that adds per-row access control.</description>
     <properties>
-        <product.version>0.1.0</product.version>
+        <product.version>0.1.1</product.version>
```

Comment:
Please increase to 0.2.0. This is a breaking change, so it would actually justify increasing the major number. But I don't want to do this until we reach production readiness.

Reply:
done